x/estaking/modules/distribution/module.go

```diff
@@ -102,15 +104,16 @@ func FilterDenoms(coins sdk.Coins, denoms ...string) sdk.Coins {
 }
 
 // AllocateTokens handles distribution of the collected fees
-func (am AppModule) AllocateTokens(ctx sdk.Context) {
+// Handles Eden and Usdc rewards
+func (am AppModule) AllocateEdenUsdcTokens(ctx sdk.Context) {
```

Comment:
Let's add more details. like USDC and Eden is distributed for staking Elys and locking Eden and locking EdenB